.env.example

```diff
@@ -332,3 +332,7 @@ STORY_PRIVATE_KEY= # Story private key
 STORY_API_BASE_URL= # Story API base URL
 STORY_API_KEY= # Story API key
 PINATA_JWT= # Pinata JWT for uploading files to IPFS
+
+# Cosmos
+COSMOS_RECOVERY_PHRASE= # 12 words recovery phrase (need to be in quotes, because of spaces)
+COSMOS_CHAIN_NAME= # Chain name
```

Comment:
It should be removed as it is not currently in use. First supported chain from character settings is selected as default/starting one.